lib/resty/auto-ssl/utils/start_sockproc.lua

```diff
@@ -5,7 +5,7 @@ local function start()
   local exit_status = os.execute("umask 0022 && " .. auto_ssl.lua_root .. "/bin/resty-auto-ssl/start_sockproc")
   -- Lua 5.2+ returns boolean. Prior versions return status code.
   if exit_status == 0 or exit_status == true then
-    local _, set_err, set_forcible = ngx.shared.auto_ssl:set("sockproc_started", true)
+    local _, set_err, set_forcible = ngx.shared.auto_ssl_settings:set("sockproc_started", true)
     if set_err then
       ngx.log(ngx.ERR, "auto-ssl: failed to set shdict for sockproc_started: ", set_err)
     elseif set_forcible then
```

Comment:
In the error message below, I think we'll just want to updated it to reference the `auto_ssl_settings` dict (instead of `auto_ssl`).